npm/xpath/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@cypress/xpath",
+  "version": "0.0.0-development",
+  "description": "Adds XPath command to Cypress.io test runner",
+  "main": "scripts",
+  "scripts": {
+    "test": "node ../../scripts/cypress.js run --e2e",
+    "cy:open": "node ../../scripts/cypress.js open --e2e --project ${PWD}",
```

Comment:
nit: name these cypress:open and cypress:run to align with other cypress test scripts in the repo